strawberryfields/configuration.py

```diff
@@ -218,6 +220,49 @@ def parse_environment_variable(key, value):
 
     return value
 
+def store_account(authentication_token, filename="config.toml", location="user_config", **kwargs):
+    """Stores an account in a configuration file.
```

Comment:
This should perhaps be modified to emphasize that this is for storing Xanadu cloud credentials primarily, since this is more user facing i.e., "Configure Strawberry Fields for access to the Xanadu cloud platform by saving your account credentials"